capsules/extra/src/atecc508a.rs

```diff
@@ -1016,7 +1016,7 @@ impl<'a> I2CClient for Atecc508a<'a> {
             Operation::ShaEnd(run) => {
                 if status == Err(i2c::Error::DataNak) || status == Err(i2c::Error::AddressNak) {
                     // The device isn't ready yet, try again
-                    if run == 50 {
+                    if run == 500 {
```

Comment:
Why? Why was it working before but not now? Is this just a magic number that happens to sort of work?  The datasheet seems to indicate a specific amount of time to wait until the device is ready (~600us it looks like)